src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -1,22 +1,40 @@
 package dev.j3fftw.litexpansion;
 
-import dev.j3fftw.litexpansion.armor.ElectricChestplate;
-import dev.j3fftw.litexpansion.items.FoodSynthesizer;
-import dev.j3fftw.litexpansion.utils.Constants;
-import dev.j3fftw.litexpansion.weapons.NanoBlade;
-import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
 import org.bukkit.Sound;
+import org.bukkit.block.Block;
 import org.bukkit.enchantments.Enchantment;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.event.entity.EntityDamageByEntityEvent;
 import org.bukkit.event.entity.EntityDamageEvent;
 import org.bukkit.event.entity.FoodLevelChangeEvent;
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.event.player.PlayerInteractEvent;
 import org.bukkit.inventory.ItemStack;
 
+import dev.j3fftw.litexpansion.armor.ElectricChestplate;
+import dev.j3fftw.litexpansion.items.DiamondDrill;
+import dev.j3fftw.litexpansion.items.FoodSynthesizer;
+import dev.j3fftw.litexpansion.items.MiningDrill;
+import dev.j3fftw.litexpansion.utils.Constants;
+import dev.j3fftw.litexpansion.utils.Utils;
+import dev.j3fftw.litexpansion.weapons.NanoBlade;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Rechargeable;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import me.mrCookieSlime.Slimefun.cscorelib2.protection.ProtectableAction;
+
 public class Events implements Listener {
 
+    ArrayList<Material> drillableBlocks = new ArrayList<>(Arrays.asList( Material.STONE, Material.ANDESITE, Material.DIORITE, Material.GRANITE ));
```

Comment:
Why just overworld blocks?

Reply:
das tru. what other blocks? end stone netherrack... basalt?

Reply:
Everything a iron pick except ores could break for tier 1 and the highest tier everything a diamond pick could break?

Reply:
include post 1.14 blocks?

Reply:
What do you mean

Reply:
basalt, blackstone

Reply:
Is that even in minecraft if so yes include does but you need to check minecraft version

Reply:
or i could use collection and check if each material name contains the word but idk about the efficiency for that

not even sure about the efficiency for the current arraylist that im using, too many cant be good